source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -1018,6 +1018,7 @@ void Graph::setConstant(UiNodePtr node, mx::InputPtr& input, const mx::UIPropert
             ImGui::PushItemWidth(-100);
             if (ImGui::Button("Browse"))
             {
+                _fileDialogInput = input;
```

Comment:
I think this should be cleared after the file dialog is used in case there are any side-effects with leaving an old input reference around.

Also could we name this something like `_fileDialogImageInput` to make it a bit clearer it's associated with the image dialog. 

Thanks.